pkg/namesgenerator/names-generator_test.go

```diff
@@ -33,13 +25,3 @@ func TestNameRetries(t *testing.T) {
 	}
 
 }
-
-// To be awesome, a container name must involve cool inventors, be easy to remember,
-// be at least mildly funny, and always be politically correct for enterprise adoption.
-func isAwesome(name string) bool {
```

Comment:
LOL, didn't know we had this one, pretty old too https://github.com/docker/docker/commit/07180f3aa75377b4c3227da006df00ec4f032589